client/src/App.js

```diff
@@ -1,20 +1,36 @@
 import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
 
 import "./App.css";
+import AuthProvider from "./utils/AuthProvider";
+import AuthPage from "./pages/AuthPage";
 
 function App() {
-  return (
-    <MuiThemeProvider theme={theme}>
-      <BrowserRouter>
-        <Route path="/" component={LandingPage} />
-      </BrowserRouter>
-    </MuiThemeProvider>
-  );
+    return (
+        <MuiThemeProvider theme={theme}>
+            <AuthProvider>
+                <BrowserRouter>
+                    <Switch>
+                    <Route exact path="/" component={LandingPage} />
+                      <Route
+                          exact path="/login"
+                      >
```

Comment:
let's use prettier.io - it'll make it easier to work together and make the code all look consistent

Reply:
just added a prettierrc.json at the root file can be used to standardize formatting for the whole project